### DISCUSSION TITLE
Training of column text in text recognition tasks

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1720] [General]
In the training of text recognition model, when the dataset contains column text, can it be input directly for training?

**Comment by Harold-lkk** [comment]
It should be rotated 90 degrees anticlockwise usually if the reading order is from top to bottom

**Reply by htaoruan** [reply]
Thank you, however, the text is not fully corrected after the counterclockwise rotation, will it affect the model effect
